### DISCUSSION TITLE
Readme night mode

### DISCUSSION THREAD

**Discussion by Oneiros90** [verdaccio/verdaccio#3467] [Q&A] [answered]
Hi, I'm not sure if this is considerable a bug but the README markdown file of the packages is not showed correctly in the web interface if the night mode is active: I see white background and black text, which is not the correct style for the night mode.
Is it a bug or just a missing feature?
Thanks

**Comment by juanpicado** [comment]
I think this sounds as improvement, initially readme was not considered as part of the dark mode.

**Comment by juanpicado** [comment, marked as answer]
https://github.com/verdaccio/verdaccio/pull/4071
